_pages/tools/channels/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Channels
+---
+
+## Channels
+
+- [#tts-email-crosspost](https://gsa-tts.slack.com/messages/tts-email-crosspost/) - TTS-wide announcements from e-mails
+- [#news](https://gsa-tts.slack.com/messages/news/) - [Updates from Outreach]({{site.baseurl}}/office-of-operations/outreach/#how-to-make-tts-wide-announcements)
+
+- Teams
+
+  - 18F - [\#news-18f](https://gsa-tts.slack.com/messages/news-18f/) for all 18F-level announcements
```

Comment:
What do you think of making the 18F link the same as Solutions, PIFs, and OA, linking to the main office channel instead of the news channel? I think either one is fine, but I'm a sucker for consistency. 🙂 

```suggestion
  - [18F](https://gsa-tts.slack.com/messages/18f/)
```